lesson_11/test_homework_11.py

```diff
@@ -0,0 +1,48 @@
+import logging
+
+from assertpy import assert_that
+from lesson_11.log_event_fun import log_event
+from logging import config
+
+
+user_name="testuser"
+
+logging.config.fileConfig('logging_config.ini')
+logging.getLogger('sampleLogger')
+
+def logger_message_parser() -> str:
+    with open("login_system.log") as f:
+        log_string = f.readlines()[-1]
+    return log_string
+
+def verify_status_and_log_level_values(log_level, status):
+    for item in [log_level, status]:
+        is_true = item in logger_message_parser()
```

Comment:
Variable is better to name as is_item_in_logger_message instead of generic is_true

Reply:
Thanks for the review. I changed variable in the following commit.